src/components/Blockquote/Blockquote.jsx

```diff
@@ -2,17 +2,28 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './Blockquote.scss';
 
-function Blockquote( { children } ) {
-	return (
-		<blockquote className={styles.blockquote}>{children}</blockquote>
-	);
+function Blockquote( { children, innerHtml } ) {
+	if ( innerHtml ) {
+		return (
+			<blockquote
+				className={styles.blockquote}
+				dangerouslySetInnerHTML={{ __html: innerHtml }}
+			/>
+		);
+	}
+
+	return <blockquote className={styles.blockquote}>{children}</blockquote>;
 };
 
 Blockquote.propTypes = {
 	/**
 	 * Child nodes to be rendered as the inner content of the blockquote.
 	 */
 	children: PropTypes.node,
+	/**
+	 * Html from a WP content block to be rendered as the inner content of the blockquote.
+	 */
+	innerHtml: PropTypes.string,
```

Comment:
This is a use-case that's pretty specific to qz.com, and given that `dangerouslySetInnerHTML` is a React anti-pattern, I think we should keep this out of Prism. Perhaps in qz-react you could `dangerouslySetInnerHTML` on an unstyled `p` or `div` and wrap that in `Blockquote`?